slices/gawk.yaml

```
@@ -0,0 +1,19 @@
+package: gawk
+
+slices:
+  libs:
+    essential:
+      - libmpfr6_libs
+      - libreadline8_libs
+      - libsigsegv2_libs
+    contents:
+      /usr/lib/x86_64-linux-gnu/gawk/*.so:
```

```suggestion
      /usr/lib/*-linux-*/gawk/*.so:
```

Otherwise, it won't work for other arch.